visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts

```diff
@@ -505,7 +505,7 @@ describe("codeMapMouseEventService", () => {
 		})
 
 		it("should add property node", () => {
-			codeMapBuilding.setNode(undefined)
+			codeMapBuilding.setNode(CODE_MAP_BUILDING.node)
```

Comment:
Test bug fix

Reply:
`codeMapBuidling` is `CODE_MAP_BUILDING` already.

```suggestion
```

Reply:
The test is somewhat redundant. I could either remove the test or make it a tad stricter.

Reply:
I just read the test for the highlighting. Those are horrible.
The test I marked here should be removed and the tests should be adapted in the future. They're not really describing what is actually expected and are not testing the actual behavior.
Do you want to remove this test and add a TODO?